test/integration/029_docs_generate_tests/test_docs_generate.py

```diff
@@ -1567,7 +1567,7 @@ def expected_seeded_manifest(self, model_database=None, quote_model=False):
                         'macros': [],
                         'nodes': ['model.test.model', 'model.test.second_model']
                     },
-                    'description': 'A description of the complex exposure',
+                    'description': 'A description of the complex exposure\n',
```

Comment:
I feel like I saw a way to add an ignore-whitespace instruction somewhere, but am doing this for now